python/webhooks_v5/create_webhook.py

```diff
@@ -0,0 +1,80 @@
+import os
+import asana
+from asana.rest import ApiException
+from dotenv import load_dotenv
+from pprint import pprint
+
+# Load environment variables (for stored PAT and X-Hook-Secret)
+load_dotenv()
+
+# Configure Asana client
+configuration = asana.Configuration()
+configuration.access_token = os.getenv('PAT')
+api_client = asana.ApiClient(configuration)
+
+# Create an instance of the Webhooks API
+webhooks_api_instance = asana.WebhooksApi(api_client)
+
+# Helper function to read X-Hook-Secret from the .env file
+def get_x_hook_secret():
+    with open('.env', 'r') as file:
+        for line in file:
+            if line.startswith('X_HOOK_SECRET='):
+                return line.split('=')[1].strip()
+    return ""
+
+# Function to delete the webhook
+# Documentation: https://github.com/Asana/python-asana/blob/v5.0.11/docs/WebhooksApi.md#delete_webhook
+def delete_webhook(webhook_gid):
+    try:
+        api_response = webhooks_api_instance.delete_webhook(webhook_gid)
```

Comment:
It doesn't look like we are using this assigned variable `api_response` so you can just call `webhooks_api_instance.delete_webhook(webhook_gid)`